app/Http/routes.php

```diff
@@ -35,7 +35,7 @@
 // Nomination
 Route::post('nomination', ['as' => 'nomination.create', 'uses' => 'NominationController@store']);
 // Admin
-Route::group(['before' => 'role:administrator', 'prefix' => 'admin'], function () {
```

Comment:
@DFurnes hmm okay apparently filters DO still exist, but this one wasn't working? I was able to access admin pages as a non-admin. In Jubilee 🎉  checking that a user is an admin is done via middleware too it looks like